reV/supply_curve/sc_aggregation.py

```diff
@@ -0,0 +1,1057 @@
+# -*- coding: utf-8 -*-
+"""reV supply curve aggregation framework.
+
+Created on Fri Jun 21 13:24:31 2019
+
+@author: gbuster
+"""
+from concurrent.futures import as_completed
+import h5py
+import logging
+import numpy as np
+import os
+import pandas as pd
+from scipy.spatial import cKDTree
+from warnings import warn
+
+from reV.handlers.outputs import Outputs
+from reV.handlers.exclusions import ExclusionLayers
+from reV.supply_curve.aggregation import (AbstractAggFileHandler,
+                                          AbstractAggregation)
+from reV.supply_curve.exclusions import FrictionMask
+from reV.supply_curve.points import SupplyCurveExtent
+from reV.supply_curve.point_summary import SupplyCurvePointSummary
+from reV.utilities.execution import SpawnProcessPool
+from reV.utilities.exceptions import (EmptySupplyCurvePointError,
+                                      OutputWarning, FileInputError,
+                                      InputWarning, SupplyCurveInputError)
+
+logger = logging.getLogger(__name__)
+
+
+class SupplyCurveAggFileHandler(AbstractAggFileHandler):
+    """Simple framework to handle aggregation file context managers."""
```

Comment:
Modify doc string to differentiate from abstract version.

Reply:
done